webclient/src/store/server/server.interfaces.ts

```diff
@@ -3,24 +3,26 @@ import { Log, SortBy, User, UserSortField } from 'types';
 export interface ServerConnectParams {
   host: string;
   port: string;
-  user: string;
-  pass: string;
+  userName: string;
+  password: string;
 }
 
 export interface ServerRegisterParams {
   host: string;
   port: string;
-  user: string;
-  pass: string;
-  passAgain: string;
```

Comment:
Are we still checking that passwords match when registering?

Reply:
Yes, but that will be validated locally in the component, no need to pass the second password up the chain.